src/vehicles/components/ScooterSheet.tsx

```diff
@@ -30,11 +33,71 @@ export const ScooterSheet = ({vehicle, close}: Props) => {
   const operatorName =
     getTextForLanguage(vehicle.system.operator.name.translation, language) ??
     t(ScooterTexts.unknownOperator);
-  const operatorAppUri =
+  const appStoreUri =
+    Platform.OS === 'ios'
+      ? vehicle.system.rentalApps?.ios?.storeUri
+      : vehicle.system.rentalApps?.android?.storeUri;
+  const rentalAppUri =
     Platform.OS === 'ios'
       ? vehicle.rentalUris?.ios
       : vehicle.rentalUris?.android;
 
+  const appStoreOpenError = (operatorName: string) => {
```

Comment:
I'm not too happy about how this code turned out. Improvement suggestions are appreciated.